README.md

```diff
@@ -28,15 +37,48 @@ import yt_idefix
 
 Now `yt.load` will be able to read Pluto/Idefix output files.
 
-## Supported formats
+### Dataset's Metadata
+The metadata is automaticly parsed from data file, definitions header file and inifile when loading dataset.
 
-| Code   | format | supported since | additional dependencies |
-|--------|:------:|:---------------:|-------------------------|
-| Idefix | `.dmp` | v0.1.0          |                         |
-| Idefix | `.vtk` | v0.3.0          |                         |
-| Pluto  | `.vtk` | v0.9.0          |                         |
-| Pluto  |  XDMF  | v1.1.0          | `h5py`                  |
+definitions header file (`definitions.h` for Pluto, or `definitions.hpp` for Idefix) and inifile (`.ini` file) are assumed to be along with data file. Their paths can also be specified explicitly with paramerters `definitions_header` and `inifile`.
```

Comment:
```suggestion
Definitions header file (`definitions.h` for Pluto, or `definitions.hpp` for Idefix) and inifile (`pluto.ini` and `idefix.ini` respectively) are discovered automatically if they match default names, and are located along with data files. Otherwise, they can be specified explicitly as paths (either relative to data files or absolute paths) with parameters `definitions_header` and `inifile` respectively.
```